go.mod

```diff
@@ -93,9 +93,9 @@ require (
 	golang.org/x/mod v0.10.0 // indirect
 	golang.org/x/net v0.10.0 // indirect
 	golang.org/x/oauth2 v0.6.0 // indirect
-	golang.org/x/sys v0.8.0 // indirect
-	golang.org/x/term v0.8.0 // indirect
-	golang.org/x/text v0.9.0 // indirect
+	golang.org/x/sys v0.13.0 // indirect
```

Comment:
Hey @rvanderp3 thanks for the PR!
Does `go mod tidy` break if we get `golang.org/x/net@v0.17.0` across the board?
What is forcing `v0.13.0`?

Reply:
hey, sorry i missed this.  i don't think it does but I was trying to directly address the dependency mentioned in the bug.

Reply:
Doing go replace is kind of a risky move in the long term, I'd be better to see if we can bump the dep that actually requires the `golang.org/x/net v0.10.0 // indirect` dep and bump that by transitivity. But this could work too.

Reply:
i can look in to that today. on it.

Reply:
i took a look at this and it looks like there are a few modules depending on this:
```
$ go mod graph | grep golang.org/x/net@v0.10
github.com/openshift/machine-api-provider-nutanix golang.org/x/net@v0.10.0
github.com/onsi/ginkgo/v2@v2.9.5 golang.org/x/net@v0.10.0
github.com/onsi/gomega@v1.27.7 golang.org/x/net@v0.10.0
github.com/openshift/machine-api-operator@v0.2.1-0.20230524144558-2430daf07c93 golang.org/x/net@v0.10.0
golang.org/x/net@v0.10.0 golang.org/x/crypto@v0.14.0
golang.org/x/net@v0.10.0 golang.org/x/sys@v0.13.0
golang.org/x/net@v0.10.0 golang.org/x/term@v0.13.0
golang.org/x/net@v0.10.0 golang.org/x/text@v0.13.0
golang.org/x/tools@v0.9.1 golang.org/x/net@v0.10.0
sigs.k8s.io/controller-runtime@v0.15.0 golang.org/x/net@v0.10.0
golang.org/x/crypto@v0.14.0 golang.org/x/net@v0.10.0
```
I'm not sure of the impact of bumping the controller runtime in particular.  Joel took a look at the 4.13 counterpart of this https://github.com/openshift/machine-api-provider-nutanix/pull/86 .  I'm fine to merge this as is unless you feel that we need to bump the rest.

Reply:
Ok I see that would involve too much. SGTM then. Thanks for checking things for me :)